doc/changelog/02-specification-language/10985-about-arguments.rst

```diff
@@ -0,0 +1,3 @@
+- :cmd:`About` and :cmd:`Print` display an :cmd:`Arguments` invocation
+  to describe argument metadata instead of prose (`#10985
+  <https://github.com/coq/coq/pull/10985>`_, by Gaëtan Gilbert).
```

Comment:
IMO that's too terse, and doesn't notify the user of a change in output which has also happened.

Reply:
"instead of" clearly means there's a change.

Reply:
I'm OK if you like it this way; I tend to signal "user-visible" changes a bit more boldly, such as:

> The output of the `Print` and `About` commands has changed. Arguments meta-data for constants is now displayed as the corresponding `Argument` command instead of the human-targeted prose used in previous Coq versions.

Anyways it would be great if we had a mechanism to put some kind of tag like in OCaml to signal "this change may break user scripts" [ @Zimmi48 ]